scripts/mirror.js

```diff
@@ -584,9 +652,18 @@ const bumpVersion = (data, destination, source, originalData) => {
  * @param {string} browser
  * @param {string} source
  * @param {string} modify
+ * @param {string} targetVersion
  @ @returns {Identifier}
  */
-const doSetFeature = (data, newData, rootPath, browser, source, modify) => {
+const doSetFeature = (
+  data,
+  newData,
+  rootPath,
+  browser,
+  source,
+  modify,
+  targetVersion,
```

Comment:
💡 Refactoring opportunity for the future: Functions with this many parameters could be refactored to accept an object instead. There's probably an ESLint rule to enforce this. 🤔